includes/components/class-admin.php

```diff
@@ -1867,4 +1904,30 @@ function( $config ) {
 			]
 		);
 	}
+
+	/**
+	 * Update dismissed HivePress pointers.
+	 */
+	public function update_dismiss_pointers() {
+
+		// Get pointer.
+		$pointer = sanitize_text_field( hp\get_array_value( $_POST, 'pointer' ) );
+
+		if ( ! current_user_can( 'edit_others_posts' ) || ! $pointer ) {
```

Comment:
leave these for admins only

Reply:
There is already checking for the admins with the current_user_can